example/spec/parallel/moveGranuleCollections/ChangeGranuleCollectionS3.js

```diff
@@ -56,69 +59,89 @@ describe('when ChangeGranuleCollectionS3 is called', () => {
     }
   });
   beforeAll(async () => {
-    const inputPayloadFilename = './spec/parallel/ingestGranule/IngestGranule.input.payload.json';
-    const providersDir = './data/providers/s3/';
-    const s3data = [
-      '@cumulus/test-data/granules/MOD09GQ.A2016358.h13v04.006.2016360104606.hdf.met',
-      '@cumulus/test-data/granules/MOD09GQ.A2016358.h13v04.006.2016360104606.hdf',
-      '@cumulus/test-data/granules/MOD09GQ.A2016358.h13v04.006.2016360104606_ndvi.jpg',
-    ];
-
-    const collectionsDir = './data/collections/s3_MOD09GQ_006_full_ingest';
-    const targetCollectionsDir = './data/collections/s3_MOD09GQ_007_full_ingest_move';
-    const granuleRegex = '^MOD09GQ\\.A[\\d]{7}\\.[\\w]{6}\\.006\\.[\\d]{13}$';
-    config = await loadConfig();
-    stackName = config.stackName;
-    const testId = createTimestampedTestId(stackName, 'IngestGranuleSuccess');
-    const testSuffix = createTestSuffix(testId);
-    testDataFolder = createTestDataPath(testId);
+    try {
+      const inputPayloadFilename = './data/payloads/IngestGranule.input.payload.json';
```

Comment:
```suggestion
      const inputPayloadFilename = './spec/parallel/ingestGranule/IngestGranule.input.payload.json';
```
where did this file path come from?

Reply:
ah, actually I see your comment in the other PR